vnext/Microsoft.ReactNative/Views/RawTextViewManager.cpp

```diff
@@ -90,7 +91,9 @@ void RawTextViewManager::NotifyAncestorsTextChanged(ShadowNodeBase *nodeToUpdate
       } else if (!std::wcscmp(nodeType, L"RCTVirtualText") && textTransform == TextTransform::Undefined) {
         textTransform = static_cast<VirtualTextShadowNode *>(parent)->textTransform;
       }
+
       parent = static_cast<ShadowNodeBase *>(host->FindShadowNodeForTag(parent->GetParent()));
+      isNested = true;
```

Comment:
We seem to always set this after the while loop - what does this variable track exactly?

Reply:
It tracks whether the Run that's being updated is the direct child of the TextBlock, in which case its value is "lifted" into the Text property to enable text optimization. It's intended to be true after the first pass through the loop.

Reply:
Sounds good